core/test/unit/api/v2/utils/serializers/output/posts_spec.js

```diff
@@ -10,9 +10,7 @@ describe('Unit: v2/utils/serializers/output/posts', function () {
 
     beforeEach(function () {
         postModel = (data) => {
-            return {
-                toJSON: sandbox.stub().returns(data)
-            };
+            return Object.assign(data, {toJSON: sandbox.stub().returns(data)});
```

Comment:
@kirrg001 had to change this implementation due to https://travis-ci.org/gargol/Ghost/jobs/442078681. We now pass `model.id` to url generating functions. So just having `toJSON` on the object didn't behave enough like a real model :)